conf/default.yaml

```diff
@@ -18,6 +18,16 @@ plugins:
   hapi-auto-handler:
     _priority: -1
   hapi-log-response:
+  ./plugins/loadHelpers:
+    _priority: -2
+    opens:
+      enabled: true
+      type: 'open'
+    clicks:
+      enabled: true
+      type: 'click'
+      tags: 'app:example-app'
+    trackingUrl: 'http://micro-metrics.docker.gcow.us'
```

Comment:
we probably shouldn't be hard coding stuff like this in a public repo. lets make this read from an env var